app/views/api/v2/import_puppetclasses/show.json.rabl

```diff
@@ -25,3 +26,7 @@ end
 node(:removed_environment, :if => ->(environment) { @changed['obsolete'][environment.name].present? && @changed['obsolete'][environment.name].match(/_destroy_/) }) do |environment|
   environment.name
 end
+
+node(:ignored_environment, :if => ->(environment) { @changed['ignored'][environment.name].present? && @changed['ignored'][environment.name].match(/_ignored_/) }) do |environment|
```

Comment:
https://github.com/theforeman/foreman/pull/4012/files#diff-fda05e73ec11be4e3f4c60d62f9bd9e7R26 fixed here, GitHub isn't smart enough to notice yet